src/containers/SiteContainer.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { BrowserRouter as Router, Link, Route } from 'react-router-dom';
+import LandingPage from '../components/LandingPage';
+import HomePage from '../pages/HomePage';
+import AboutPage from '../pages/AboutPage';
+import ContactUsPage from '../components/ContactUsPage';
+import NavBar from '../components/NavBar';
+import '../styling/site-container-styling.css';
+
+class SiteContainer extends React.Component {
+  state = {};
+
+  // contains the routing information for the site
+  render() {
+    return (
+      <div>
+        <Router>
+          <Route path="/" exact={true} render={() => <LandingPage />} />
+
+          <Route
+            path="/home"
+            exact={true}
+            render={() => (
+              <div>
+                <NavBar />
+                <HomePage />
+              </div>
+            )}
+          />
+
+          <Route
+            path="/about"
+            exact={true}
+            render={() => (
+              <div>
+                <NavBar />
+                <AboutPage />
+              </div>
+            )}
+          />
+
+          <Route
+            path="/contact"
+            exact={true}
+            render={() => (
+              <div>
+                <NavBar />
+                <ContactUsPage />
+              </div>
+            )}
+          />
+        </Router>
+      </div>
+    );
+  }
+}
+
+export default SiteContainer;
```

Comment:
What is the purpose of this SiteContainer and do we really need it? I'm pretty sure I removed this page and refactored all of the routing to go into App.js so this page isn't necessary anymore - maybe you were just refactoring from an outdated dev branch? If so, could you delete this file please?

Reply:
Yes looks like the outdated container, @daniaalnahdi if you delete the file it should not cause any issues on the site!